Makefile

```diff
@@ -285,6 +285,7 @@ ifeq ($(shell [[ $(OPENSHIFT_CI) == "true" && $(REPO_NAME) == "assisted-service"
 	@cd assisted-service && \
 	git fetch origin pull/$(PULL_NUMBER)/head:assisted-service-pr-$(PULL_NUMBER) && \
 	git checkout assisted-service-pr-$(PULL_NUMBER)
+	git rebase origin/$(PULL_BASE_REF)
```

Comment:
You'll want ``PULL_BASE_REF := $(or ${PULL_BASE_REF}, "")`` at the top of the makefile (like with ``PULL_NUMBER`` and others)
In addition, adding it here: https://github.com/openshift/release/blob/9eb742d4427e133e7c62a8170efb6acf0e64ace1/ci-operator/step-registry/assisted/baremetal/setup/assisted-baremetal-setup-commands.sh#L103